Assorted doc bug fixes

jacobhammons opened the pull request:
Refs #4381
Refs #8146
Refs #13036
Refs #6853
Refs #9319
Refs #13777
Refs #2229
Refs #14946
Refs #15042

jacobhammons committed:
Assorted doc bug fixes

Refs #4381
Refs #8146
Refs #13036
Refs #6853
Refs #9319
Refs #13777
Refs #2229
Refs #14946
Refs #15042

rallytime commented on doc/ref/configuration/master.rst:
```diff
@@ -2668,6 +2668,32 @@ This can be used to control logging levels more specifically. See also
+    'value': 27, 'description': 'EVENT_DISCONNECTED'}
+
+All events logged will include the string ``ZeroMQ event``. A connection event
+should be logged on the as the minion starts up and initially connects to the
```
Missing a word here `logged on the as`

rallytime commented on doc/ref/configuration/master.rst:
```diff
@@ -2668,6 +2668,32 @@ This can be used to control logging levels more specifically. See also
+Default: ``False``
+
+To diagnose issues with minions disconnecting or missing returns, ZeroMQ
+supports the use of monitor sockets # to log connection events. This
```
Was the `#` intended here?

jacobhammons committed:
Updated `zmq_monitor` docs

jacobhammons commented:
@rallytime I fixed both the typos in the explanation, I also realized that I put this explanation in the master config, not the minion, so I corrected that too. Thanks for reviewing.

rallytime commented:
@jacobhammons Oh, good catch. I didn't notice that. Looks good!
